domainservice/src/main/kotlin/ua/marchenko/artauction/auction/mapper/AuctionEventProtoMapperExtension.kt

```diff
@@ -0,0 +1,34 @@
+package ua.marchenko.artauction.auction.mapper
+
+import com.google.protobuf.Timestamp as TimestampProto
+import java.time.LocalDateTime
+import java.time.ZoneOffset
+import ua.marchenko.artauction.auction.domain.AuctionCreatedEvent
+import ua.marchenko.artauction.auction.model.MongoAuction
+import ua.marchenko.internal.output.pubsub.auction.AuctionCreatedEvent as AuctionCreatedEventProto
+
+fun MongoAuction.toAuctionCreatedEventProto(): AuctionCreatedEventProto {
+    return AuctionCreatedEventProto.newBuilder().also { builder ->
```

Comment:
apply?

Reply:
it may look better here, but I use `also` in my other mappers, so I'm keeping it to match the style across the project